corehq/apps/case_importer/views.py

```diff
@@ -292,11 +279,33 @@ def bulk_case_upload_api(request, domain, **kwargs):
         error = get_importer_error_message(e)
     except SpreadsheetFileExtError:
         error = "Please upload file with extension .xls or .xlsx"
-        return json_response({'error_msg': _(error)})
+    return json_response({'code': 500, 'message': _(error)}, status_code=500)
 
 
 def _bulk_case_upload_api(request, domain):
-    upload_file, case_type, search_field, create_new_cases, search_column =_get_bulk_case_upload_args_from_request(request, domain)
+    try:
+        upload_file = request.FILES["file"]
+        case_type = request.POST["case_type"]
+        if not upload_file or not case_type:
+            raise Exception
+    except Exception:
```

Comment:
nit: could be a bit more succinct and avoid catching `Exception` (not a best practice because it could catch unexpected errors that should not be silenced).
```suggestion
    upload_file = request.FILES.get("file")
    case_type = request.POST.get("case_type")
    if not upload_file or not case_type:
```